vignettes/FAQ.Rmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "rbmi: Frequently Asked Questions"
+author: Alessandro Noci, Craig Gower-Page and Marcel Wolbers
+output: 
+  bookdown::html_document2:
+    toc: true
+    toc_depth: 4
+    number_sections: true
+    citation_package: natbib
+    base_format: rmarkdown::html_vignette
+bibliography: "references.bib"
+link-citations: true
+pkgdown:
+  as_is: true
+linkcolor: blue
+vignette: >
+  %\VignetteIndexEntry{rbmi: Frequently Asked Questions}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+    collapse = TRUE,
+    comment = "#>"
+)
+```
+
+# Introduction
+
+This document provides answers to common questions about the `rbmi` package.
+It is intended to be read after the `rbmi: Introduction` vignette.
```

Comment:
```suggestion
It is intended to be read after the `rbmi: Quickstart` vignette.
```

Not sure the "Introduction" vignette exists, maybe you are referring to the "Quickstart"?